static/xml/toolbox.xml

```diff
@@ -452,6 +452,15 @@
                 </block>
                 <block type="loader"></block>
                 <block type="block_holder"></block>
+                <block type="webhook">
+                    <value name="WEBHOOK_URL">
+                        <shadow type="text">
```

Comment:
`shadow` can be changed into `block` as the user is not able to connect other blocks if it's a shadow type.